src/pages/restaurantPage/RestaurantPage.js

```diff
@@ -0,0 +1,71 @@
+//Libraries
+import React from 'react';
+
+//Constants
+import {
+  INITIAL_STATE,
+  RESTAURANT_ID,
+  DEFAULT_ERROR_MESSAGE,
+} from './constants/restaurantPage.general';
+
+//Helpers
+import fetchRestaurantData from '../../api/fetchRestaurantData';
+import restaurantPageReader from './readers/restaurantPageReader';
+
+//Components
+import Loader from '../../commonComponents/loader';
+import ErrorHandler from '../../commonComponents/errorHandler';
+import RestaurantDetails from '../../components/restaurantDetails';
+import Menu from '../../components/menu';
+
+class RestaurantPage extends React.Component {
+  state = INITIAL_STATE;
+  componentDidMount() {
+    this.getRestaurantData();
+  }
+  getRestaurantData() {
+    fetchRestaurantData(RESTAURANT_ID)
+      .then(this.setRestaurantPageState)
+      .catch(this.handleError)
+      .finally(this.setAsLoaded);
+  }
+  setRestaurantPageState = (restaurantData) => {
+    const dishes = restaurantPageReader.dishes(restaurantData);
```

Comment:
the function name is setRestaurantPageState but it is using reader and setting state. so either change the name of function or create a new function which handles the reader part and then calls this function